frontend_app_html-document/src/component/HtmlDocument.jsx

```diff
@@ -69,25 +89,41 @@ export const HtmlDocument = props => {
             onClickBtn={props.onClickShowDraft}
           />
         )}
-
         {(props.mode === APP_FEATURE_MODE.VIEW || props.mode === APP_FEATURE_MODE.REVISION) && (
           <div>
-            <div className='html-document__contentpage__textnote__version'>
-              {props.t(
-                'Version #{{versionNumber}}', {
-                  versionNumber: props.mode === APP_FEATURE_MODE.VIEW && !props.isRefreshNeeded
-                    ? props.lastVersion
-                    : props.version
-                }
+            <div className='html-document__contentpage__textnote__top'>
+              {props.translationState === TRANSLATION_STATE.PENDING && (
+                <span className='html-document__contentpage__textnote__top__translation'>
+                  <i className='fa fa-spinner fa-spin' /> {props.t('Translation pending…')}
```

Comment:
Why `props.t('Translation pending…')` is not at `TOGGLE_TRANSLATION_TEXT[TRANSLATION_STATE.PENDING]` and here you can use `TOGGLE_TRANSLATION_TEXT[props.translationState]` ?

Reply:
`toggleTranslationText` is used for knowing which text to put on the translation *button*. I extended it to `null` for PENDING/DISABLED tests mainly for convenience.
This text is only a loader and not (directly) related to the button allowing to translate.